src/opm/parser/eclipse/share/keywords/000_Eclipse100/W/WCONPROD

```diff
@@ -2,13 +2,13 @@
       [{"name" : "WELL"         , "value_type" : "STRING"},
        {"name" : "STATUS"       , "value_type" : "STRING" , "default" : "OPEN"},
        {"name" : "CMODE"        , "value_type" : "STRING"},
-       {"name" : "ORAT"         , "value_type" : "DOUBLE",  "default" : 0.0 , "dimension" : "LiquidSurfaceVolume/Time"},
-       {"name" : "WRAT"         , "value_type" : "DOUBLE" , "default" : 0.0 , "dimension" : "LiquidSurfaceVolume/Time"},
-       {"name" : "GRAT"         , "value_type" : "DOUBLE" , "default" : 0.0 , "dimension" : "GasSurfaceVolume/Time"},
-       {"name" : "LRAT"         , "value_type" : "DOUBLE" , "default" : 0.0 , "dimension" : "LiquidSurfaceVolume/Time"},
-       {"name" : "RESV"         , "value_type" : "DOUBLE" , "default" : 0.0 , "dimension" : "LiquidSurfaceVolume/Time"},
-       {"name" : "BHP"          , "value_type" : "DOUBLE" , "default" : 1.01325 , "dimension" : "Pressure"},
-       {"name" : "THP"          , "value_type" : "DOUBLE" , "default" : 0.0 , "dimension" : "Pressure"},
+       {"name" : "ORAT"         , "value_type" : "UDA",  "default" : 0.0 , "dimension" : "LiquidSurfaceVolume/Time"},
+       {"name" : "WRAT"         , "value_type" : "UDA" , "default" : 0.0 , "dimension" : "LiquidSurfaceVolume/Time"},
+       {"name" : "GRAT"         , "value_type" : "UDA" , "default" : 0.0 , "dimension" : "GasSurfaceVolume/Time"},
+       {"name" : "LRAT"         , "value_type" : "UDA" , "default" : 0.0 , "dimension" : "LiquidSurfaceVolume/Time"},
+       {"name" : "RESV"         , "value_type" : "UDA" , "default" : 0.0 , "dimension" : "LiquidSurfaceVolume/Time"},
+       {"name" : "BHP"          , "value_type" : "UDA" , "default" : 1.01325 , "dimension" : "Pressure"},
```

Comment:
Here too.  1.01325 looks a lot like 1 atmosphere in bars.  Is this default value safe for all unit systems?